src/otx/core/model/utils/mmpretrain.py

```diff
@@ -48,3 +54,132 @@ def create_model(config: DictConfig, load_from: str | None = None) -> tuple[nn.M
     """
     classification_layers = get_classification_layers(config, MODELS, "model.")
     return build_mm_model(config, MODELS, load_from), classification_layers
+
+
+class ForwardExplainMixInForMMPretrain(Generic[T_OTXBatchPredEntity, T_OTXBatchDataEntity]):
+    """OTX classification model which can attach a XAI hook."""
```

Comment:
```suggestion
    """Mix-in class to support XAI feature for MM Pretrained models commonly.

    This is a mix-in class that cannot be used as standalone. The correct usage of it will be like
    ```python
    class MobileNetV3ForMulticlassCls(ForwardExplainMixInForMMPretrain, MMPretrainMulticlassClsModel):
    ```
    """
```